visualizations/vis_class.py

```diff
@@ -0,0 +1,24 @@
+from collections.abc import MutableMapping
+from abc import abstractmethod
+
+
+class Visualization(MutableMapping):
+    @abstractmethod
+    def __init__(self, filename):
+        pass
+
+    @abstractmethod
+    def _import_file(self):
+        """
+        This method takes a JSON file and returns a dictionary of data for the \
```

Comment:
I don't think it `takes a JSON file` as much as it will use the `file_name` field to access a JSON file, etc.